examples/generate_QG_time_series.py

```diff
@@ -0,0 +1,84 @@
+"""Truth-twin and observation time series for QG (quasi-geostrophic) model."""
+
+import numpy as np
+import scipy.ndimage.filters as filters
+from matplotlib import pyplot as plt
+from dapper.mods.QG import square
+import dapper.tools.progressbar as pb
+from dapper.mods.QG.sakov2008 import HMM
+from dapper.mods.QG.sakov2008 import obs_inds
+
+###########
+# Auxiliary plotting function
+###########
+
+def show(x0, psi=True, ax=None):
+    #
+    def psi_or_q(x):
+        return x if psi else compute_q(x)
```

Comment:
`compute_q` not imported

Reply:
Cleaning up extra definitions, unused functions, this included.